### PR TITLE
US6 // parts - add newPart

### PR DESCRIPTION
**Pull request by matschi3**
Hi ✌🏼
i also edited my header stuff so it is a little bit more.

[US6](https://github.com/matschi3/capstone/issues/6)
[Deployment](https://capstone-hxn79hgfe-matschi3.vercel.app/)

Note: 
-remember your choosen category to find your new created Part
-img will be default for now

### CODE REVIEW

**Review by actyvystom — CHANGES_REQUESTED** [review]
Great work! 🚀 
I like that you are using `zustand` 🙂 
See one tiny comment on mixing import syntax.

Cheers, Thomas

File: components/PartForm/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { categories } from "@/lib/categories";
+import { FormContainer, Label, Input, Select } from "./PartForm.styled";
+import { PartCardFlexContainer } from "../PartCard/PartCard.styled";
```
[the remark this answers is not in the document]
Comment (matschi3) [reply components/PartForm/index.js]: Thanks. good advice, i will fix and sty consistent ;)

**Comment by matschi3** [comment]
Thank you @actyvystom for your time to review my monster US 🙏
`zustand` is great! yesterday i played around with state and came up to the point i kind of need sth like it to suit my case.

🥳

File: components/LinkBack/LinkBack.styled.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import { StyledLink } from "../../components/StyledLink/StyledLink.styled.js";
+
+export const StyledLinkBack = styled(StyledLink)`
```
Comment (Airglow98) [inline components/LinkBack/LinkBack.styled.js:4]: Why did you styled your StyledLink instead of putting these propertys directly to your StyledLink?

**Review by Airglow98 — COMMENTED** [review]
Nice progress. I wrote you a comment to the StyledLink please check it.  Rest looks good to me.

File: components/LinkBack/LinkBack.styled.js (hunk 2 above)
Comment (matschi3) [reply components/LinkBack/LinkBack.styled.js:4]: im using `StyledLink` to actually style my links for now.
`StyledLinkBack` + `StyledLinkTo` im using for positioning and giving different props :)

**Comment by matschi3** [comment]
Thanks for your time ✌🏼

**Pull request merged**